library/core/class.form.php

```diff
@@ -243,14 +243,23 @@ public function calendar($FieldName, $Attributes = false) {
     /**
      * Returns Captcha HTML & adds translations to document head.
      *
+     * Events: BeforeCaptcha
+     * 
      * @return string
      */
     public function captcha() {
+        $handled = false;
+        $this->EventArguments['Handled'] =& $handled;
+        $this->fireEvent('BeforeCaptcha');
```

Comment:
Because the primary purpose of this event is to replace rather than prep, I think it would be appropriate to simply call this event `'Captcha'`.